typhon/tests/arts/test_griddedfield.py

```diff
@@ -285,6 +285,13 @@ def test_deepcopy(self):
         # Grids should not be the same object.
         assert a.grids is not b.grids
 
+    def test_from_xarray(self):
+        a = xml.load(self.ref_dir + 'GriddedField3.xml').copy()
```

Comment:
You don't need to call `.copy()` here. Maybe it is better to remove it to prevent future confusion